apps/bluetooth-music/app.js

```diff
@@ -41,12 +45,29 @@ function onAudioFocusLost (transient, mayDuck) {
       needResume = false
     }
   } else {
+    nowPlayingCenter.setNowPlayingInfo(null)
+    nowPlayingCenter.removeListener('command', onAudioFocusRemoteCommand)
```

Comment:
I don't think removing listener is necessary for the case of bluetooth music since there is no multiple player instance to be controlled and command would not be received on null playing info.

Reply:
I just copy code from cloud-player, could you tell me whats diff between them?

Reply:
The raison d'êter of these statements in cloud-player is the multiple instances of player to be controlled by the media controller.

Reply:
You say "command would not be received on null playing info." And cloud player also call `nowPlayingCenter.setNowPlayingInfo(null)`, so what's the diff of multiple instances?

Reply:
Setting playing info as `null` is application wide, which makes the application not receiving the command. Yet while one of the player does want to receive the command in cloud-player and one another don't, it's the most simplest way to replace the listeners.

Reply:
Though I still cannot understand what you said, but I will remove it now first.
And If there's necessary of register listener by `on` but  do not need to unregister it. I think the better way is to pack `setNowPlayingInfo` and `register listerer` together.

Reply:
In short, it's just not _necessary_ to remove the listeners in the case, but it would work like a charm in both ways. 

And as I indicated before, `setNowPlayingInfo` _could_ be called multiple times during one event `command` listening.

Reply:
So, there's no different result between remove listener and don't remove listener, the code is not wrong, just only redundant?

Reply:
Yes.